src/main/java/scm/address/model/file/Filename.java

```diff
@@ -7,10 +7,13 @@
  * Guarantees: immutable; filename is valid as declared in {@link #isValidFilename(String)}
  */
 public class Filename {
-    public static final String MESSAGE_CONSTRAINTS = "Filenames should be alphanumeric.";
+    public static final String MESSAGE_CONSTRAINTS = "Filenames should be alphanumeric and .json or .csv files. "
+            + "Example: 'myContacts.json' or 'myContacts2.csv'";
 
     public static final String MESSAGE_DUPLICATE = "Filenames should be unique.";
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}_\\-]+";
+
+    public static final String VALIDATION_REGEX = "\\p{Alnum}+\\.json|\\p{Alnum}+.csv";
```

Comment:
It would be good to put the two groups into parentheses.

Reply:
Done!